.github/workflows/publish.yml

```diff
@@ -0,0 +1,59 @@
+name: Publish to PyPI
+on:
+  release:
+    # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release
+    types: [released]
+jobs:
+  test:
+    uses: ./.github/workflows/test.yml
+
+  build:
+    needs: ['test']
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+
+      - name: Install Node dependencies
+        run: |
+          npm ci
+
+      - name: Compile JavaScript
+        run: |
+          npm run build
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+
+      - name: Install Python dependencies
+        run: pip install build
+
+      - name: Build package distributions
+        run: python -m build
+
+      - uses: actions/upload-artifact@v3
+        with:
+          path: ./dist
+
+  # https://docs.pypi.org/trusted-publishers/using-a-publisher/
+  pypi-publish:
+    needs: ['build']
+    environment: 'publish'
```

Comment:
Not sure, the example is without quotes.

https://docs.pypi.org/trusted-publishers/using-a-publisher/

Reply:
That should make no difference, but fun read: https://norwayyaml.com/